kakao/KakaoLoginIOS/KakaoLoginIOS/MainViewController.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  MainViewController.swift
+//  KakaoLoginIOS
+//
+//  Created by Kevin Kang on 2017. 9. 18..
+//  Copyright © 2017년 Hara Kang. All rights reserved.
+//
+import FirebaseAuth
+
+class MainViewController: UIViewController {
+    @IBOutlet weak var profileImageView: UIImageView!
+    @IBOutlet weak var emailTextView: UITextView!
+    @IBOutlet weak var displayNameTextView: UITextView!
+    @IBOutlet weak var navigationBar: UINavigationBar!
+    @IBOutlet weak var navigationTitleItem: UINavigationItem!
+    
+    override func viewDidLoad() {
+        let user = Auth.auth().currentUser
+        if user == nil {
+            self.dismiss(animated: true, completion: {
+            })
+        } else {
+            updateUserProfileUI(user: user!)
+        }
+    }
+    
+    /**
+        Update profile UI with user information stored on Firebase (email, display name, and profile image)
+    */
+    func updateUserProfileUI(user: User) {
+        self.emailTextView.text = String.init(format: "Email: %@", user.email!)
```

Comment:
nit: Initializers don't need to explicitly reference init. (Prefer `String(format:)` over `String.init(format:)`)